src/main/java/org/apache/mesos/hdfs/config/SchedulerConf.java

```diff
@@ -34,6 +34,10 @@ public String getMesosDnsDomain() {
     return getConf().get("mesos.hdfs.mesosdns.domain", "mesos");
```

Comment:
Could you update the default values in schedulerConf to mirror you changes to cpu and memory in mesos-site.xml?